clients/src/main/resources/common/message/EnvelopeRequest.json

```diff
@@ -16,10 +16,11 @@
 {
   "apiKey": 58,
   "type": "request",
-  "listeners": ["controller"],
+  "listeners": ["controller", "zkBroker"],
   "name": "EnvelopeRequest",
   // Request struct for forwarding.
-  "validVersions": "0",
+  // Version 1 adds the envelope request support to ZK brokers as well.
+  "validVersions": "0-1",
```

Comment:
hmm, I don't think this needs a version bump.

Even if we did the version bump, if a ZK-based broker sent a version 0 request it would work fine, right?